code/modules/mob/living/simple_animal/hostile/hellbound.dm

```diff
@@ -0,0 +1,75 @@
+/mob/living/simple_animal/hellbound
+	name = "damned spirit"
+	real_name = "damned spirit"
+	desc = "Your eternal reward"
+	icon = 'icons/mob/mob.dmi'
+	icon_state = "shade"
+	icon_living = "shade"
+	maxHealth = INFINITY
+	health = INFINITY
+	speak_emote = list("whines")
+	emote_hear = list("wails.","screeches.")
+	response_help  = "puts their hand through"
+	response_disarm = "flails at"
+	response_harm   = "punches"
+	friendly = "stares sullenly at"
+	speak_chance = 1
+	melee_damage_lower = 0
+	melee_damage_upper = 0
+	attacktext = "sighs at"
+	minbodytemp = 0
+	maxbodytemp = INFINITY
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	speed = 10
+	stop_automated_movement = 1
+	status_flags = GODMODE
+	faction = list("hellbound")
+	status_flags = CANPUSH
+	flying = 1
+	var/damned_key = null
+
+/mob/living/simple_animal/hellbound/Life()
+	..()
+	if(damned_key && ckey != damned_key)
+		ckey = damned_key
+		src << "Where do you think you're going?"
+
+/mob/living/simple_animal/hellbound/death()
+	return
+
+/mob/living/simple_animal/hellbound/suicide()
+	return
+
+/mob/living/simple_animal/hellbound/ghostize(can_reenter_corpse = 1)
+	return
```

Comment:
apart from this just being a bad idea in general wouldn't this make admins unable to aghost if in this state?